index.js

```diff
@@ -201,39 +204,23 @@ function tcurl(opts) {
         }
 
         if (err) {
-            console.error(err);
-            console.error(err.message);
+            display('error', err);
+            display('error', err.message);
             /*eslint no-process-exit: 0*/
             process.exit(1);
         }
 
         if (!resp.ok) {
-            console.error('Got call response not ok');
+            display('error', 'Got call response not ok');
```

Comment:
Errors don’t have to be JSON encoded on stderr. We can just use console.error for this case.